### DISCUSSION TITLE
Accessibility on Card - navigating through list of cards using keys - Tab, Arrow [when CardHeader not used], Tooptip for card

### DISCUSSION THREAD

**Discussion by kmanisha29** [UI5/webcomponents-react#6113] [Q&A]
If we have a list of cards in Flex layout, can we navigate through keys and does UI5 card provide accessibility for card list to navigate through each card and click action on the card if FlexBox is being used. Any list layout provided by UI5 which is accessible?
Currently I had few extra things to be added in content as header so need to use custom component in CardHeader and I see that only card header was accessible and selected with tab but not card. Do we need any additional configuration?
Also, Is there a way to provide tooltip to card
https://stackblitz.com/edit/github-savnpj?file=src%2FApp.tsx,src%2FCustomizedCard.tsx,src%2FUICard.tsx,src%2FCardHeader.tsx,src%2Fdata.ts

**Comment by MarcusNotheis** [comment]
Hi @kmanisha29,
this discussion sounds pretty much like the issue you've opened. Is it about the same issue?
Then I would recommend we'll close this discussion and follow up in https://github.com/SAP/ui5-webcomponents/issues/9557.

**Comment by kmanisha29** [comment]
Yes its same issue.
